client/state/ui/editor/video-editor/actions.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_EDITOR_RESET_STATE,
+	VIDEO_EDITOR_SET_POSTER_URL,
+	VIDEO_EDITOR_SHOW_ERROR,
+	VIDEO_EDITOR_SHOW_UPLOAD_PROGRESS,
+	VIDEO_EDITOR_UPDATE_POSTER,
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used for resetting the video editor state.
+ *
+ * @return {Object} Action object
+ */
+export const resetState = () => {
+	return {
+		type: VIDEO_EDITOR_RESET_STATE,
+	};
+};
```

Comment:
fun fact! we can use arrow functions without `{}`s here since all we're doing is returning a value

```js
export const resetState = () => ( {
	type: VIDEO_EDITOR_RESET_STATE,
} );
```

notice that because _objects_ and _function bodies_ share the curlies, I had to surround the returned value in parentheses. this is the only time the arrow-function syntax demands them